app/controllers/boards_controller.rb

```diff
@@ -180,7 +179,13 @@ def og_data
     }
   end
 
+  def include_creator
+    return unless params.key?(:board) && params[:board].key?(:writer_ids)
+    return if params[:board][:writer_ids].include?(@board.creator_id)
+    params[:board][:writer_ids] << @board.creator_id
```

Comment:
does this do anything funny if you submit an invalid update and it renders the page with the submitted params?

Reply:
Nope, but also you inspired me to refactor to not need this hack